wandb/data_types.py

```diff
@@ -905,7 +908,8 @@ def to_json(self, run):
         if self._boxes:
             json_dict['boxes'] = self._boxes.to_json(run)
         if self._masks:
-            json_dict['masks'] = [mask.to_json(run) for mask in self._masks]
+            json_dict['masks'] = {
+                    k: mask.to_json(run) for (k, mask) in self._masks.items()}
```

Comment:
I still think we should store the mask as image data.  I believe @shawnlewis agreed on the image approach.  I guess I could be swayed but it just feels really inefficient to store all the mask data in json, there will be no compression at all...

Reply:
The mask is using image data.

The `to_json` call creates a file for the mask data:

https://github.com/wandb/client/pull/883/files